src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/parser/LagartoBasedHtmlParser.java

```diff
@@ -181,8 +181,6 @@ public void tag(Tag tag) {
                 break;
             case END:
                 break;
-            default:
```

Comment:
This is a switch over `jodd.lagarto.TagType` enum. The current switch covers all enum cases.
I think it would be better to ensure enum switches are exhaustive rather than fail in runtime.